app/models/raised_indicator.rb

```diff
@@ -12,11 +12,16 @@
 #  updated_at         :datetime         not null
 #  tag_id             :integer
 #  participant_uuid   :string
+#  uuid               :uuid
+#  selected           :boolean          default(FALSE)
 #
 class RaisedIndicator < ApplicationRecord
   include Tagable
 
   belongs_to :scorecard, foreign_key: :scorecard_uuid, optional: true
   belongs_to :indicatorable, polymorphic: true
   belongs_to :tag, optional: true
+  has_one    :voting_indicator, foreign_key: :raised_indicator_uuid, primary_key: :uuid
```

Comment:
@hengsokly ideally, raised indicator that is selected should have more than one voting(participant 1, raised_indicator_selected_1, score), could you please check this?

Reply:
Yes, I'm wrong. Actually, voting_indicator(selected indicator) should have many raised_indicators